read.go

```diff
@@ -376,8 +371,16 @@ func readBytes(r dicomio.Reader, t tag.Tag, vr string, vl uint32) (Value, error)
 
 func readString(r dicomio.Reader, t tag.Tag, vr string, vl uint32) (Value, error) {
 	str, err := r.ReadString(vl)
-	// String may have '\0' suffix if its length is odd.
-	str = strings.Trim(str, " \000")
+	onlySpaces := true
+	for _, char := range str {
+		if !unicode.IsSpace(char) {
+			onlySpaces = false
+		}
+	}
+	if !onlySpaces{
+		// String may have '\0' suffix if its length is odd.
+		str = strings.Trim(str, " \000")
+	}
```

Comment:
I made this change because in one test case the value for "Referring Physician Name" was just two spaces "  " and this was trimmed down to an empty string by the existing code block. @suyashkumar I'd like to hear your thoughts on how to navigate this case

Reply:
Probably the best way to handle this is to only trim \000 and not trim the space character. (e.g. just remove the space in front of \000

Reply:
I tries this as a fix as well before resorting to my current solution and for some reason the two spaces still got trimmed down to the empty string

Reply:
Gotcha. If you happen to know of an open source / openly available DICOM that has this behavior, it might be good for us to include in a test set. If so, if you could open up a separate issue and attach the dicom there, that would be awesome!